.perfer/yoshi.config.js

```diff
@@ -0,0 +1,33 @@
+const path = require('path');
```

Comment:
The solution of creating bundles for the components is inspired by wix-style-react